### DISCUSSION TITLE
Xbox Controller

### DISCUSSION THREAD

**Discussion by AlperenBaris** [ndeadly/MissionControl#787] [Q&A]
Can i  connect XBOX Wireless 9. Gen controller? I confused description on README about Xbox Controllers.

**Comment by ndeadly** [comment]
I don't know what you mean by 9 gen. Only model 1708 (Xbox One X|S) controllers on the legacy Bluetooth firmware can be connected with the release version. More recent models or controllers updated to the BLE firmware don't work yet, but there are experimental test builds available on my discord server that are capable of connecting them.
